.gitignore

```diff
@@ -243,3 +243,5 @@ TSWLatexianTemp*
 
 # generated if using elsarticle.cls
 *.spl
+
+.DS_Store
```

Comment:
Good idea ... but you should do this on an Ubuntu machine ... not a Mac ...

Reply:
Hehe yes. Main PC (this I'm writing this comment on) is an Ubuntu. Mac is the side laptop. Don't worry I'm a Linux guy ;)